api/tests/unit/domain/models/CertificationCandidate_test.js

```diff
@@ -229,20 +229,48 @@ describe('Unit | Domain | Models | Certification Candidate', function () {
     });
 
     context('when isCertificationBillingEnabled feature toggle is on', function () {
-      it('should throw an error if billingMode is null', async function () {
-        // given
-        sinon.stub(featureToggles, 'isCertificationBillingEnabled').value(true);
-        const certificationCandidate = domainBuilder.buildCertificationCandidate({
-          billingMode: null,
+      context('when the certification center is SCO and managing students (billingMode is null)', function () {
```

Comment:
```suggestion
      context('when the certification center is SCO and not managing students (billingMode is null)', function () {
```
On pourrait rajouter en plus le test pour du sco managing students
```
      context('when the certification center is SCO and managing students (billingMode is null)', function () {
```